requirements.txt

```diff
@@ -4,6 +4,7 @@ Flask-Admin==1.1.0
 Flask-RESTful==0.3.5
 Flask-Login==0.3.2
 Flask-OAuthLib==0.9.2
+Flask-SQLAlchemy==2.1
```

Comment:
Re: alchy, it seems the author's mostly moved onto https://github.com/dgilland/sqlservice#history